freetextresponse/freetextresponse.py

```diff
@@ -322,13 +362,12 @@ def _get_indicator_class(self):
         """
         Returns the class of the correctness indicator element
         """
-        result = ''
-        if self.count_attempts == 0:
-            result = 'unanswered'
-        elif self._determine_credit() == Credit.zero:
-            result = 'incorrect'
-        else:
-            result = 'correct'
+        result = 'unanswered'
+        if self.display_correctness and self._word_count_valid():
```

Comment:
@caseylitton Maybe we're talking about different things?

If `min_words` is 5, but my submitted response is only 3 words, my response is `incorrect`.
But the way I'm reading this logic, you only ever display "correct" or "incorrect" if
`min <= len(response) <= max`, right?

Or am I misreading this?